src/components/QrScanner/types.ts

```diff
@@ -0,0 +1,9 @@
+import { Dispatch, SetStateAction } from "react";
+import { Attendee } from "../RegistrationTable/columns";
+
+export interface QrProps {
+  event: { id: string; year: string };
+  rows?: Attendee[];
```

Comment:
I'm not sure why `rows` is optional?

Reply:
old codebase didn't have rows passed as a prop but with how we're implementing it now it doesnt make sense anymore. Thanks!